docs/skinning.md

```diff
@@ -0,0 +1,111 @@
+# Skinning Your uPortal-home #
+
+_Prerequisites:_
+
+_A working uPortal-app-framework application._
+
+_A code editor._
+
+For a more in-depth discussion of theming, see https://github.com/UW-Madison-DoIT/uw-frame/blob/master/docs/theming.md
+
+1. Locate and open the frame-config.js file in your uw-frame-components/js/ directory.
+
+
+2. Increment the themeVersion in line 9. 
```

Comment:
I believe due to front end style caching issues.  The ui doesn't try to fetch theming changes unless this number changes